docker/FileCheckTests/compile_time_patterns

```diff
@@ -0,0 +1,4 @@
+; COM: Checks the pattern of compile time log file. Makes sure the header
```

Comment:
Can we move this test outside of the docker directory? The usual convention for LLVM-based compiler projects is to put unit-tests under `unit` or `units` and FileCheck/LIT tests under `test/tests`. So I'm thinking something like `test/check_compile_time_log.txt`

Reply:
sure.